services/core-web/src/utils/Validate.js

```diff
@@ -50,6 +50,9 @@ export const Validate = new Validator();
 
 export const required = (value) => (value ? undefined : "This is a required field");
 
+export const requiredList = (value) =>
+  value && value.length > 0 ? undefined : "This is a required field";
+
```

Comment:
Created to validate fields with list data types

Reply:
Does the `required` validation not work in this situation?

Reply:
I tried that one first and realized it wasn't working because an empty array is considered "truthy" in JS.

Reply:
I could probably change "required" to 
`export const required = (value) => (value && value.length > 0 ? undefined : "This is a required field");` and it could work for both string and array/object fields.